Dockerfile

```diff
@@ -10,7 +10,7 @@ FROM $DOCKER_ARCH/centos:7 AS relay-deps
 ARG DOCKER_ARCH
 ARG BUILD_ARCH=x86_64
 # Pin the Rust version for now
-ARG RUST_TOOLCHAIN_VERSION=1.63.0
+ARG RUST_TOOLCHAIN_VERSION=1.64.0
```

Comment:
@tonyo should we revert #1482 instead? Or is the best way forward to keep pinning this?

Reply:
I'd suggest to keep pinning it manually until there's an automation (e.g. a GitHub Action) in place to check for the latest Rust version and open a PR with version bumps.
If we revert #1482, we'll have the same problem in the future that we tried to fix there: the Rust toolchain Docker layer will be cached